tests/scripts/thread-cert/backbone/test_bmlr.py

```diff
@@ -0,0 +1,197 @@
+#
+#  Copyright (c) 2020, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the name of the copyright holder nor the
+#     names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS 'AS IS'
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+# This test verifies that PBBR sends BMLR.ntf correctly when multicast addresses are registered.
+#
+# Topology:
+#    ---- -(eth)-------
+#       |        |
+#      PBBR----SBBR
+#        \     /
+#        Router1---Commissioner
+#
+
+import unittest
+
+import thread_cert
+from pktverify.packet_verifier import PacketVerifier
+
+PBBR = 1
+SBBR = 2
+ROUTER1 = 3
+COMMISSIONER = 4
+
+REREG_DELAY = 4  # Seconds
+MLR_TIMEOUT = 300  # Seconds
+CUSTOM_MLR_TIMEOUT = 1000  # Seconds
+
+MA1 = 'ff04::1'
+MA2 = 'ff04::2'
+MA3 = 'ff04::3'
+MA4 = 'ff04::4'
+MA5 = 'ff04::5'
+
+
+class BBR_5_11_01(thread_cert.TestCase):
+    USE_MESSAGE_FACTORY = False
+
+    TOPOLOGY = {
+        PBBR: {
+            'name': 'PBBR',
+            'allowlist': [SBBR, ROUTER1],
+            'is_otbr': True,
+            'version': '1.2',
+            'router_selection_jitter': 1,
+        },
+        SBBR: {
+            'name': 'SBBR',
+            'allowlist': [PBBR, ROUTER1],
+            'is_otbr': True,
+            'version': '1.2',
+            'router_selection_jitter': 1,
+        },
+        ROUTER1: {
+            'name': 'ROUTER1',
+            'allowlist': [PBBR, SBBR, COMMISSIONER],
+            'version': '1.2',
+            'router_selection_jitter': 1,
+        },
+        COMMISSIONER: {
+            'name': 'COMMISSIONER',
+            'allowlist': [PBBR, SBBR, ROUTER1],
```

Comment:
should match the topology diagram
```suggestion
            'allowlist': [ROUTER1],
```